apis/config/v1alpha1/types_certmanager.go

```diff
@@ -0,0 +1,34 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Certmanager specifies cluster-wide settings for cert-manager.
+type Certmanager struct {
```

Comment:
`CertManager`

Reply:
that's actually a good point and needs to be changed in quite some places :+1: